### PR TITLE
Gh652 2

### PR DESCRIPTION
**Pull request by u-fischer**
This is alternative (!) pull request for issue 652. 

# Internal housekeeping

## Status of pull request

- Feedback wanted 
- Ready to merge

## Checklist of required changes before merge will be approved
- [x] Test file(s) added (not for the actual issue which needs hyperref but new test file for fleqn added)
- [x] Version and date string updated in changed source files
- [x] Relevant `\changes` entries in source included
- [x] Relevant `changes.txt` updated
- [ ] `ltnewsX.tex` (and/or `latexchanges.tex`) updated:  needed?

### CODE REVIEW

File: required/amsmath/testfiles/amsldoc-4-fleqn.lvt [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+pmatrix
+\begin{equation}\label{eq:D}
+\begin{pmatrix} D_1t&-a_{12}t_2&\dots&-a_{1n}t_n\\
+-a_{21}t_1&D_2t&\dots&-a_{2n}t_n\\
+\hdotsfor[2]{4}\\
+-a_{n1}t_1&-a_{n2}t_2&\dots&D_nt\end{pmatrix},
+\end{equation}
+
+\end{document}
```
Comment (FrankMittelbach) [inline required/amsmath/testfiles/amsldoc-4-fleqn.lvt:53]: missing newline at end of file, doesn't hurt but shows up on other platforms.

**Review by FrankMittelbach — APPROVED** [review]
looks ok to me but misses a paragraph in ltnews34

File: required/amsmath/testfiles/amsldoc-4-fleqn.lvt (hunk 1 above)
Comment (u-fischer) [reply required/amsmath/testfiles/amsldoc-4-fleqn.lvt:53]: ok then I will delete the other PR and write a paragraph for this one.  But I'm not sure I know what to do about the newline. What does show up here?

Comment (FrankMittelbach) [reply required/amsmath/testfiles/amsldoc-4-fleqn.lvt:53]: you don't need to drop the pull requests you can do all the updates in the same one this way we don't generate unnecessary new issues. in the end we can squash all updates into a single merge. As for the missing newline I can fix that if you don't see it. The last line of the file has no newline at the end and editors like emacs flag that as in some programs that can be harmful (TeX doesn't mind)

Comment (FrankMittelbach) [reply required/amsmath/testfiles/amsldoc-4-fleqn.lvt:53]: Screenshot 2021-08-25 at 00 24 44

SourceTree also warns :-)

Comment (FrankMittelbach) [reply required/amsmath/testfiles/amsldoc-4-fleqn.lvt]: I'll fixed it

**Comment by car222222** [comment]
what happened to the idea of adding some useful indirection here?

**Pull request merged**